components/layout/footer/static-google-map.tsx

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react';
+import {StorageType} from '@/types/storage';
+
+// v2
+
+type MapProps = {
+  storage: StorageType;
+};
+
+export default function Map({ storage }: MapProps) {
+  return (
+    <a className="w-full h-28" href={`http://maps.google.com/?q=1200${storage.location}`}>
+      <img className="w-full h-full"
```

Comment:
Use NextJS Image instead of img, also make sure it's sized correctly and optimized, here's some docs to help you with that: https://nextjs.org/docs/app/api-reference/components/image

Reply:
thx for the documentation